tests/auth_tests/test_models.py

```diff
@@ -194,6 +194,18 @@ def test_last_login_default(self):
         user2 = User.objects.create_user(username='user2')
         self.assertIsNone(user2.last_login)
 
+    def test_user_clean_normalize_email(self):
+        user = User(username='user', password='foo', email='foo@BAR.com')
+        user.clean()
+        self.assertEqual(user.email, 'foo@bar.com')
+
+    def test_user_clean_normalize_username(self):
```

Comment:
not needed -- this is covered by `test_clean_normalize_username` added in the other patch